src/sage/doctest/forker.py

```diff
@@ -553,6 +553,8 @@ def __init__(self, *args, **kwds):
         self.total_walltime_skips = 0
         self.total_performed_tests = 0
         self.total_walltime = 0
+        if sys.version_info.minor < 13:
```

Comment:
```suggestion
        if sys.version_info < (3,13):
```
seems cleaner and more future proof ???

Reply:
Fair enough, but I hope these if's are gone by the time we reach Python 4.

Reply:
Most likely it will. Unless you add a `sage: assert sys.version_info.major < (4.11)  # when this test break, delete the hack in this function!!!` in the doctest.